tests/MeiliSearchBoxTest.js

```diff
@@ -17,7 +18,11 @@ describe('MeiliSearchBox', () => {
       '<form id="search-form" role="search" class="meilisearch-search-wrapper search-box">'
     )
     expect(wrapper.html()).toContain(
-      '<input id="meilisearch-search-input" class="search-query" placeholder="">'
+      '<input id="meilisearch-search-input" class="search-query dsb-input" placeholder="" autocomplete="off" spellcheck="false" role="combobox" aria-autocomplete="list" aria-expanded="false" aria-label="search input" aria-owns="meilisearch-autocomplete-listbox-0" style="vertical-align: top;" dir="auto">'
```

Comment:
Why are these being added?

Reply:
Because otherwise the test would fail. The search input has these attributes added by docs-searchbar.js

Reply:
Why is it that the tests passed before?

Reply:
It actually never did. Executing the test on the main branch with node 12 I get:
```
yarn run v1.22.19
$ vue-cli-service test:unit
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
(node:17740) UnhandledPromiseRejectionWarning: Error: Error: No input element in the page matches #meilisearch-search-input
(node:17740) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:17740) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
 PASS  tests/MeiliSearchBoxTest.js
  MeiliSearchBox
    ✓ renders default form and input (31ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        1.368s
Ran all test suites.
✨  Done in 4.00s.
```

As you can see the Vue component silently fails to mount (with node >= 14 it errors out), so when the test checks the HTML it is not changed by docs-searchbar.js.  With this PR it successfully mounts the component and executes docs-searchbar.js, resulting in changed HTML.

Reply:
Thanks, that's very nice.

I'm opening an issue for this silently failing test.